subsys/net/ip/net_stats.h

```diff
@@ -287,6 +287,52 @@ static inline void net_stats_update_per_proto_drop(struct net_if *iface,
 	}
 }
 
+static inline void net_stats_update_l2_proto_recv(struct net_if *iface,
```

Comment:
@tbursztyka, what would say about this? Don't we have too many stats layers. Like, L3 stats, ethernet stats, and now these L2 stats.

Reply:
I'd leave the stats out of the picture for now. (in a separate and future patch, if needed)

Reply:
Stats dropped in new version. I will work on it later.